ably/types/message.py

```diff
@@ -129,6 +129,8 @@ def from_json(obj):
         elif encoding and encoding == six.u('cipher+base64'):
             ciphertext = base64.b64decode(data)
             data = CipherData(ciphertext, obj.get('type'))
+        elif encoding and encoding == six.u('json'):
+            data = json.loads(data)
```

Comment:
This is good, however if a body is cipher+base64 and JSON, then this does not work i.e. it would need to recurse through the encoding attributes until complete.  Lets leave as is and look at that when the tests for that behaviour is added